run

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-exec gunicorn --log-level debug -w 4 -b 127.0.0.1:8000 teabot_endpoints.endpoints:app
+exec gunicorn --log-level debug -w 4 -b 0.0.0.0:${PORT:-8000} teabot_endpoints.endpoints:app
```

Comment:
gunicorn listening on all interfaces is probably okay on heroku, but is a terrible idea if we're attempting to hide it behind an nginx. https://github.com/ryandotsmith/nginx-buildpack suggests using a unix domain socket at `/tmp/nginx.socket`. If I decide to make the heroku version work with nginx, it would be useful to have a copy of your nginx config for teabot.co.uk. If not, I might just switch between public and private interfaces based on whether I think we're on heroku.